generator/common/const.py

```diff
@@ -5,9 +5,14 @@
 
 # General keys
 CONFIG_DID = "did"
+CONFIG_DID = "did"
+CONFIG_TaC = "terms-and-conditions"
```

Comment:
```suggestion
CONFIG_T_AND_C = "terms-and-conditions"
```

the lowercase character would break convention for constants, and it's called `T_AND_C` already in the `CONFIG_IAAS` case

Reply:
You are right. I changed it.